src/switch_rtp.c

```diff
@@ -4770,7 +4775,13 @@ SWITCH_DECLARE(switch_status_t) switch_rtp_set_video_buffer_size(switch_rtp_t *r
 	}
 
 	if (!max_frames || frames >= max_frames) {
-		max_frames = frames * 10;
+	    if (vb_size_factor && atoi(vb_size_factor) < MAX_VIDEO_BUFFER_SIZE_FACTOR+1 && atoi(vb_size_factor) > MIN_VIDEO_BUFFER_SIZE_FACTOR) {
```

Comment:
atoi(vb_size_factor) is used 3 times here. also the indentation is not good , eg: "MAX_VIDEO_BUFFER_SIZE_FACTOR+1" should be "MAX_VIDEO_BUFFER_SIZE_FACTOR + 1" . Again, where you added switch_core_set_variable() in switch_core.c,  it should be "switch_core_set_variable("video_jb_size_factor", val);" . yes, we're picky about these things. also, commits should be squashed into 1 commit.